pages/specification/migration/_index.md

```diff
@@ -3,5 +3,85 @@ title: Migrating from older drafts
 section: docs
 ---
 
-The release notes discuss the changes impacting users and implementers:
+### Introduction
 
+Just as your schemas evolve over time, the JSON Schema specification does as well. This guide provides useful information to make upgrading your schemas across versions of the specification a breeze.
+
+Learn how to use our migration guides and tooling ([AlterSchema](https://alterschema.sourcemeta.com/)) to bring your work with you.
+
+### Keywords Overview
+
+Here is a comprehensive overview to get you going.
```

Comment:
My version - Here is a comprehensive overview to get you **started**

I read a pointer on Google style guide to draft this way, but I'm unable to recall the exact one. Please ignore if not relevant.

Reply:
Agreed Anitha, I will use your version. Thank you.